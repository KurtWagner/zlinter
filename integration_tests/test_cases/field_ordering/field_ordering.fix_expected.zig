const MyStruct = struct {
    const decl_also = 2;
    const decl = 1;

    field_d: u32 = 0,

    /// Field with comment
    /// across multiple
    /// lines
    field_a: u32,

    /// Field with single line comment
    field_c: u32,

    fn ok() void {}
};

const MyEnum = enum {
    /// Comment on single line
    my_enum_d,
    my_enum_a,
    /// Multiline line
    /// comment for enum c
    my_enum_c,
};

const MyUnion = union {
    my_union_a: struct {
        /// Nested fields
        c: f32,
        d: u32,
        a: u32,
    },

    /// With doc comments
    /// on multiple lines
    my_union_c: f32,

    my_union_b: u32,
};

const MyError = error{
    error_d,
    /// Single line
    error_a,
    /// Multi
    /// line
    error_c,
};
