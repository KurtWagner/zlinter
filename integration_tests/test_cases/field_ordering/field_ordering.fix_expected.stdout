9 fixes applied to: .zig-cache/tmp/field_ordering.input.zig
Fixed 9 issues in 1 files!
0 issues disabled by comments.
19 fix passes remaining
1 fixes applied to: .zig-cache/tmp/field_ordering.input.zig
Fixed 1 issues in 1 files!
0 issues disabled by comments.
18 fix passes remaining
Fixed 0 issues in 0 files!
0 issues disabled by comments.
Total of 10 issues fixed
